docs/assets/templates/services/template-minimum.dockstore.yml

```diff
@@ -0,0 +1,22 @@
+version: 1.2
+service:
+  subclass: <DOCKER_COMPOSE | KUBERNETES | HELM | SWARM | NOT_APPLICABLE>
+  name: <String>
+  authors:
+    - name: <String>
+      email: <String>
+  description: <String>
+  publish: <Boolean>
+  files: <String, do not use wildcards>
+  scripts:
+    start: <String>
+  environment:
+    <String-name-of-environment-variable>:
+      default: <String or Int>
+      description: <String>
+  data:
+    <String-name-of-dataset>:
+      targetDirectory: <String, relative path>
```

Comment:
A relative path is not a type

Reply:
It's not meant to represent a type, it's meant to clarify that it needs to be a relative path, without using the comments that template-maximum relies on. Is there a better way to indicate this? Or maybe it isn't really a necessary indication.

Reply:
It is inconsistent. All the other ones (except for "Files", which I should have commented on) look like types, and then this one has an extra description.

Reply:
How about line 10?

`  files: <String, do not use wildcards>
`

I thought that was important to include, but definitely can change files and targetDirectory to just `<String>`, would that be preferred?